docs/internal.rst

```diff
@@ -156,17 +156,17 @@ correspond to?
 Each ``SNode`` can have a different virtual-to-physical mapping. ``physical_index_position[i] == -1``
 means the ``i``-th virtual index does not corrspond to any physical index in this ``SNode``.
 
-``SNode`` s in handy dense tensors (i.e., ``a = ti.var(ti.i32, shape=(128, 256, 512))``)
+``SNode`` s in handy dense fields (i.e., ``a = ti.field(ti.i32, shape=(128, 256, 512))``)
```

Comment:
```suggestion
``SNode`` s in handy dense tensors (i.e., ``a = ti.field(ti.i32, shape=(128, 256, 512))``)
```
Do not do `tensors` -> `fields` yet. We **only do `ti.var` -> `ti.field` in this PR**.